Update README.md (#4)
DevExpress-Examples/devextreme-datagrid-custom-column-chooser: README.md

[![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/T947741)
[![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
<!-- default badges end -->
# DataGrid for DevExtreme - How to implement a custom column chooser

This example demonstrates how to implement a custom column chooser for a DataGrid or TreeList. 

![Custom column chooser](/column-chooser.png)

> DataGrid and TreeList include a built-in column chooser that allows users to alter column visibility at runtime. If you need more flexible solution, implement a custom column chooser. Use the [Popup](https://js.devexpress.com/Documentation/ApiReference/UI_Widgets/dxPopup/) and [List](https://js.devexpress.com/Documentation/ApiReference/UI_Widgets/dxList/) components for this purpose.

Reviewer: ```suggestion
The DataGrid and TreeList include a built-in column chooser that allows users to alter column visibility at runtime. If you need a more flexible solution, you can use the [Popup](https://js.devexpress.com/Documentation/ApiReference/UI_Widgets/dxPopup/) and [List](https://js.devexpress.com/Documentation/ApiReference/UI_Widgets/dxList/) components to implement a custom column chooser.
```

> In this example, the custom column chooser includes a *Select All* check box that can be used to select/deselect all columns at once. All alterations in column visibility are applied on button click.

Reviewer: ```suggestion
In this example, the custom column chooser includes a *Select All* checkbox that can be used to select/deselect all columns at once. All alterations in column visibility are applied when you click the button.
```
